lucet-runtime/lucet-runtime-tests/src/guest_fault.rs

```diff
@@ -435,6 +435,7 @@ macro_rules! guest_fault_tests {
                     SigSet::all(),
                 );
                 unsafe { sigaction(Signal::SIGSEGV, &sa).expect("sigaction succeeds") };
+                unsafe { sigaction(Signal::SIGBUS, &sa).expect("sigaction succeeds") };
```

Comment:
so this is why the test's `SIGBUS` handler never appeared to be restored :grimacing:

`guest_fault.rs` becomes even more ominous on macos.

Reply:
Aw come on. Good catch

Reply:
And likewise, let's try to only modify the handler that is relevant for the current OS.

Reply:
In the test or in `lucet-runtime`? Since in the runtime we're unconditionally replacing `SIGBUS` already, I think you mean the test.

Reply:
Yes, sorry. My context for this was recommending platform-specific constants earlier in the test file, rather than checking for both.